core/node/libp2p/rcmgr_logging.go

```diff
@@ -47,11 +48,19 @@ func (n *loggingResourceManager) start(ctx context.Context) {
 			case <-ticker.C:
 				n.mut.Lock()
 				errs := n.limitExceededErrs
-				n.limitExceededErrs = 0
-				n.mut.Unlock()
-				if errs != 0 {
-					n.logger.Warnf("Resource limits were exceeded %d times, consider inspecting logs and raising the resource manager limits.", errs)
+				n.limitExceededErrs = make(map[string]int)
+
+				for e, count := range errs {
+					n.previousErrors = true
+					n.logger.Errorf("Resource limits were exceeded %d times with error %q. Consider inspecting logs and raising the resource manager limits.", count, e)
 				}
+
+				if len(errs) == 0 && n.previousErrors {
+					n.previousErrors = false
+					n.logger.Errorf("Resource limits were back to normal.")
```

Comment:
I don't know what it means for "resource limits to be back to normal".

Resource limits don't change.  Resource consumption changes.

That said, is your definition of "back to normal" meaning that in the last "time period" (I'm not sure the right term here) we no longer see resource limits being exceeded?   If so, maybe say "Resrouce limits are no longer being exceeded." ?

Also, should this be in Info instead of Error?